classes/Category.php

```diff
@@ -1049,19 +1051,20 @@ public function getProducts(
 
         if ($random === true) {
             $sql .= ' ORDER BY RAND() LIMIT ' . (int) $randomNumberProducts;
-        } else {
-            $sql .= ' ORDER BY ' . (!empty($orderByPrefix) ? $orderByPrefix . '.' : '') . '`' . bqSQL($orderyBy) . '` ' . pSQL($orderWay) . '
+        } elseif ($finalOrderBy != 'orderprice') {
```

Comment:
So there is no order when ordering by price?

Reply:
Ordering by price is at line 1065